src/gateway/ws.rs

```diff
@@ -310,11 +373,7 @@ impl WsClient {
 
         self.send_json(&WebSocketMessage {
             op: Opcode::Resume,
-            d: WebSocketMessageData::Resume {
-                session_id,
-                token,
-                seq,
-            },
+            d: WebSocketMessageData::Resume { session_id, token, seq },
```

Comment:
Unrelated formatting change. This can happen if you're running the stable version of Rustfmt, since we use a few nightly-only options.

```suggestion
            d: WebSocketMessageData::Resume {
                session_id,
                token,
                seq,
            },
```